cmd/influx/auth.go

```diff
@@ -46,6 +46,16 @@ func helpText(perm string) struct{ readHelp, writeHelp string } {
 	return help
 }
 
+func hidden(perm string) bool {
+	var hiddenTypes = map[string]struct{}{
+		"functions":    {},
```

Comment:
Hiding functions here since we haven't implemented the related CLI commands, but maybe it's ok to leave visible?

Reply:
But I could see this argued either way, so I don't feel strongly.

Reply:
We've been proactively hiding the replication commands so I think it makes sense to follow suit here (and piggy-back on the existing un-hide ticket). For functions I could see making it visible now since it's live in Cloud.